calyx-utils/Cargo.toml

```diff
@@ -12,4 +12,5 @@ serde = { workspace = true, features = ["derive"], optional = true }
 atty.workspace = true
 string-interner.workspace = true
 itertools.workspace = true
-petgraph.workspace = true
\ No newline at end of file
+petgraph.workspace = true
+symbol_table = { version = "0.3", features = ["global", "serde"] }
```

Comment:
The `serde` feature should presumably be gated to only activate when `serialize` feature is requested right?

Reply:
Oh probably, I'll fix that once I get around to looking up the proper syntax